bundler.d/development.rb

```diff
@@ -6,4 +6,5 @@
   gem 'single_test'
   gem 'pry'
   gem 'rubocop', '0.38.0' if  RUBY_VERSION > "1.9.2"
+  gem 'benchmark-ips'
```

Comment:
Added ruby-prof, but I've left the profiling script itself out (of limited use I think.)